repository_service_tuf_api/__init__.py

```diff
@@ -97,60 +100,70 @@ def sync_redis():
         redis_loader.write(settings_repository, settings_repository.to_dict())
 
 
-# Tokens
-if secrets_settings.TOKEN_KEY.startswith("/run/secrets/"):
-    try:
-        with open(secrets_settings.TOKEN_KEY) as f:
-            SECRET_KEY = f.read().rstrip("\n")
-    except OSError as err:
-        logging.error(str(err))
-
-else:
-    SECRET_KEY = secrets_settings.TOKEN_KEY
-
-if secrets_settings.ADMIN_PASSWORD.startswith("/run/secrets/"):
-    try:
-        with open(secrets_settings.ADMIN_PASSWORD) as f:
-            ADMIN_PASSWORD = f.read().rstrip("\n")
-    except OSError as err:
-        logging.error(str(err))
+SECRET_KEY: Optional[str] = None
+ADMIN_PASSWORD: Optional[str] = None
+db: Optional[sessionmaker] = None
 
-else:
-    ADMIN_PASSWORD = secrets_settings.ADMIN_PASSWORD
+if is_auth_enabled is True:
+    # Tokens
+    if secrets_settings.TOKEN_KEY.startswith("/run/secrets/"):
+        try:
+            with open(secrets_settings.TOKEN_KEY) as f:
+                SECRET_KEY = f.read().rstrip("\n")
+        except OSError as err:
+            logging.error(str(err))
```

Comment:
Maybe we can still add a small test for those lines where we have those settings set but an `OSError` is raised?

Reply:
Well, it's a little different as it's about not opening a file.
My case with `AUTH` was about not being able to run the API functions without `security`.

Reply:
Yes, but the origins of the testing issue are the same.
The `repository_service_tuf_api` is imported by the pytest already, and I cannot overwrite/mock it to properly test.
So, I cannot `pretend.raise` the open here.

Reply:
Due the complication, I filed a specific issue for it
- #338